Detectors/TPC/simulation/include/TPCSimulation/DigitContainer.h

```diff
@@ -79,16 +79,12 @@ class DigitContainer
   Sector mSector;                  ///< ID of the currently processed sector
   TimeBin mFirstTimeBin;           ///< First time bin to consider
   TimeBin mEffectiveTimeBin;       ///< Effective time bin of that digit
-  TimeBin mTmaxTriggered;          ///< Maximum time bin in case of triggered mode (TPClength / width of time bin)
+  TimeBin mTmaxTriggered;          ///< Maximum time bin in case of triggered mode (hard cut at average drift speed with additional margin)
   std::deque<DigitTime> mTimeBins; ///< Time bin Container for the ADC value
 };
 
-inline DigitContainer::DigitContainer() : mSector(-1), mFirstTimeBin(0), mEffectiveTimeBin(0), mTmaxTriggered(0), mTimeBins(500)
+inline DigitContainer::DigitContainer() : mSector(-1), mFirstTimeBin(0), mEffectiveTimeBin(0), mTmaxTriggered(550), mTimeBins(500)
```

Comment:
Hi @matthiasrichter ,
we can move this to one of the parameter classes which go to CCDB and can then also be configured via the command line. At the end the number will depend on the decision how many time bins to read in case of the triggered readout. Do you think this makes more sense?

@a-mathis , perhap you can put this to `ParameterDetector`. What do you think?

Reply:
Hi @matthiasrichter, @wiechula,
as discussed above I moved the parameter to the corresponding class
Thanks for the suggestion!

Reply:
Looks good, another option would be to set it directly in the init list
```
mTmaxTriggered(ParameterDetector::defaultInstance().getMaxTimeBinTriggered())
```
but maybe one wants to make the ParameterDetector instance configurable in the future, so its fine